### PR TITLE
Small change showing how to start PubSub

### PR DESCRIPTION
**Pull request by aphillipo**
Small change showing how to start PubSub in any old application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +17,25 @@
...
+
+
+        defmodule MyApp do
+          use Application
+        
+          def start(_type, _args) do
+            import Supervisor.Spec, warn: false
+        
+            children = [
+              supervisor(Phoenix.PubSub.PG2, [MyApp.PubSub, [pool_size: 1]])
```
Comment (Gazler) [inline README.md]: Perhaps we should leave out the `pool_size` option? It defaults to the number of schedulers now.

File: README.md [hunk 2]
```diff
@@ -18,7 +17,25 @@
         def application do
           [applications: [:phoenix_pubsub]]
         end
-
+        
+        
+## Initialisation (without Phoenix)
+
+
+        defmodule MyApp do
```
Comment (Gazler) [inline README.md]: There are too many spaces here. The indentation on the installation section is due to being in a list. 4 spaces here should be sufficient.

**Comment by Gazler** [comment]
@aphillipo Thanks! I have added a couple of line notes.

File: README.md (hunk 2 above)
Comment (binaryseed) [inline README.md]: Also, it'd be nice to get syntax highlighting with ```elixir

https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown

**Comment by chrismccord** [comment]
@aphillipo Thanks! Can you rebase from master and resolve conflicts? After that I can merge. Thanks!

**Comment by aphillipo** [comment]
I'll sort this out once I get over my Jetlag.  It's on my list 👍

**Comment by chrismccord** [comment]
ping @aphillipo :)

**Comment by aphillipo** [comment]
Ha :-) I'm in Japan 🇯🇵 I'll look at this now!

**Pull request closed without merging**